<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newnewplant</class>
 <widget class="QWidget" name="newnewplant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(184, 255, 195);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>3</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>⠀⠀⠀⣿⣿⡀
⣤⣤⣤⣿⣿⣿⣦⣤⣾⣦⣤⡀⠀⠀⠀⠀⠀⢀⣤⣤⣄
⠛⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣧⡀⠀⠀⠀⢀⡜⠛⠛⠛
⠀⠀⠀⠈⠻⣿⣿⣿⣿⡿⠋⢹⣿⡆⠀⢀⣼⣿⣿⣿⣷⣶⣶⣶
⠀⠀⠀⠀⠀⠈⠉⠉⠉⠀⠀⢸⣿⣿⣶⣿⣿⣿⣿⣿⣿⣿⠟⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠸⢿⣿⡏⠉⠙⠿⠿⠿⠟⠁
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢈⣹⡇
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⣹⣇⡀
⠀⠀⠀⠀⠀⢀⣀⣀⣀⣀⣀⣠⣾⣿⣿⣷⣶⣶⣶⣦⡀
⠀⠀⠀⢀⣴⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⠀⠀⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⠀⠀⠀⠙⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠛⠋</string>
   </property>
  </widget>
  <widget class="QLabel" name="estagioNew">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Broadway</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);
border-color: rgb(0, 85, 0);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="regarNewpushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
border-width: 1px;
border-style: solid;
color: rgb(0, 0, 127);
background-color: rgb(161, 246, 255);</string>
   </property>
   <property name="text">
    <string>REGAR</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarNew">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
